method/message/messageWitnessMessage.json

```diff
@@ -0,0 +1,38 @@
+{
+	"$schema": "http://json-schema.org/draft/2019-09/schema#",
+	"$id": "http://pop2020.dedis.ch/method/message/messageWitnessMessage.json",
+	"title": "This represenets a message someone would want to publish on a specific channel",
+	"description": "Match a message used in a \"witness a message query\"",
+	"type": "object",
+
+	"properties": {
+		"data": {
+			"description": "[String] data contained in the message",
+			"$ref": "utility/base64String.json",
+			"$comment": "Note: the string is encoded in Base64"
+		},
+		"sender": {
+			"description": "[String] public key of the witness",
+			"$ref": "utility/base64String.json",
+			"$comment": "Note: the string is encoded in Base64"
+		},
+		"signature": {
+			"description": "[String] witness' signature on data : Sign(Hash(data))",
+			"$ref": "utility/base64String.json",
+			"$comment": "Note: the string is encoded in Base64"
+		},
+		"message_id": {
+			"description": "[String] message id : Hash(data||signature)",
+			"$ref": "utility/base64String.json",
+			"$comment": "Note: the string is encoded in Base64"
+		},
```

Comment:
No need for an external ref here:
https://json-schema.org/understanding-json-schema/reference/non_json_data.html

Other than that, LGTM 👍